src/Admin/PageAdmin.php

```diff
@@ -175,7 +175,13 @@ protected function configureListFields(ListMapper $list): void
             ])
             ->add('decorate', null, ['editable' => true])
             ->add('enabled', null, ['editable' => true])
-            ->add('edited', null, ['editable' => true]);
+            ->add('edited', null, ['editable' => true])
+            ->add(ListMapper::NAME_ACTIONS, ListMapper::TYPE_ACTIONS, [
+                'translation_domain' => 'SonataAdminBundle',
```

Comment:
If I understand correctly, even with this, it displays `list.label__actions` ?

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/7926

Reply:
I think it's understandable, because
- The label depends on the LabelTranslatorStrategy
- The translation domain is messages by default, but depends on the admin configuration

So I would say that we could remove `'translation_domain' => 'SonataAdminBundle'` and we should add translations for `list.label__actions` in PageBundle.

But I wonder how we didn't need this in any other bundle so far.
The other solution would be to add the translations in SonataAdminBundle, in order to use it in any other bundle. (But technically we should support all the TranslatorStrategy in https://github.com/sonata-project/SonataAdminBundle/tree/4.x/src/Translator ...)

Reply:
SonataUserBundle needs similar solution, because is relying on addIdentifier.